source/_header.html.erb

```diff
@@ -1,15 +1,43 @@
-<header class="header">
-  <nav role="navigation" aria-label="main">
-    <ul class="header-nav container">
-      <li class="header-logo">
-        <a href="http://emberjs.com/"><span class="visually-hidden">Ember homepage</span></a>
-      <%= link_to_page "Guides", "http://guides.emberjs.com" %>
-      <%= link_to_page "API", "/api" %>
+<script type="text/javascript">
```

Comment:
might want to wrap this in a `<% content_for :foot do %>`, see https://github.com/emberjs/website/blob/master/source/about.html.erb#L5-L23